backend/typescript/services/implementations/schedulingService.ts

```diff
@@ -409,6 +459,7 @@ class SchedulingService implements ISchedulingService {
           recurring_donation_id: scheduling.recurringDonationId,
           recurring_donation_end_date: scheduling.recurringDonationEndDate,
           notes: scheduling.notes,
+          volunteer_id: scheduling.volunteerId,
```

Comment:
we dont need this either because volunteerId will not be passed at this stage when creating a donation. same for other places where this line was added in createScheduling endpoint